kubernetes/windows/hostlogs/main.ps1

```diff
@@ -86,13 +58,30 @@ Set-EnvironmentVariables
 Start-FileSystemWatcher
 
 if(Get-GenevaEnabled){
-    Invoke-Expression ".\opt\genevamonitoringagent\genevamonitoringagent\Monitoring\Agent\MonAgentLauncher.exe -useenv"
+    Write-Host "Starting Windows AMA in 1P Mode"
+
+    Start-Job -Name "WindowsHostLogsJob" -ScriptBlock { 
+        Start-Process -NoNewWindow -FilePath ".\opt\genevamonitoringagent\genevamonitoringagent\Monitoring\Agent\MonAgentLauncher.exe" -ArgumentList @("-useenv")
+    }
+
+
 } else {
-    Write-Host "Geneva not configured. Watching for config map"
-    # Infinite loop keeps container alive while waiting for config map
-    # Otherwise when the process ends, kubernetes sees this as a crash and the container will enter a crash loop
-    while($true){
-        Start-Sleep 3600
+    Write-Host "Geneva not configured. Watching for config map."
+
+    [System.Environment]::SetEnvironmentVariable("MONITORING_GCS_ENVIRONMENT", "Test" ,"Process")
```

Comment:
"Also, if I'm not mistaken, since the process scope is the default, $Env:MYVAR = "something" is functionally the same to this code while being more idiomatic in PowerShell. " 

Not sure what you are to trying say here.

Reply:
Using ```[System.Envrionment]::SetEnvironment...``` is the standard set by CI evenv for setting process level variables.

https://github.com/microsoft/Docker-Provider/blob/386c6bf0e4eb9041c5f3751e7025fbaca670d50d/kubernetes/windows/main.ps1#L43

Reply:
Moved setting the vars to the parser script